README.md

```diff
@@ -109,6 +109,13 @@ rule](https://docs.aws.amazon.com/AmazonS3/latest/userguide/lifecycle-expire-gen
 to your storage bucket if you want to retain your backups for a
 limited time, and discard backups beyond a certain age.
 
+If you are running Kubernetes in a production environment, you might need to store the 
+dump and tar files in a [generic ephemeral volume](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) instead the default local node storage
```

Comment:
```suggestion
dump and tar files in a [generic ephemeral volume](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) instead of the default local node storage
```